float32 x
float32 y
float32 z
float32 conf
uint8 class_index
